public/css/themes/SpaceGroove.css

```diff
@@ -0,0 +1,83 @@
+body {
+	color: white;
+	text-shadow: 1px 1px 12px rgb(115, 255, 255);
+	opacity: 1;
+	background-color: rgba(0, 0, 0, 0.2);
+}
+
+nav {
+	background-color: rgb(0, 0, 0, 0.2);
+}
+
+main {
+	text-shadow: 1px 1px 12px rgba(255, 255, 255, 0.699);
+}
+
+a {
+	color: rgb(255, 39, 237);
+	text-shadow: 1px 1px 12px rgb(238, 109, 255);
+}
+
+#navLeft a {
+	color: white;
+	text-shadow: 1px 1px 12px rgba(255, 255, 255, 0.699);
+}
+
+#navRight a {
+	color: white;
+	text-shadow: 1px 1px 12px rgba(255, 255, 255, 0.699);
+}
+
+#player #song, #player #artist {
+	text-shadow: 1px 1px 12px rgb(255, 255, 255);
+}
+
+div.fullscreen-bg {
```

Comment:
I feel like this style (and related) should be part of a "VideoTheme.css" sort of file that gets `@import`ed into the normal CSS file for video themes - That would reduce the total amount of labor that goes into developing a video theme (Instead of copy-pasting video theme work, you just import the correct setup for it), and it would make it easier to maintain consistency if for some outlandish reason the mechanism behind video themes needed to be updated in the future

Reply:
I will look into this.